documentation-site/examples/progress-steps/dotted.js

```diff
@@ -29,7 +35,12 @@ function ProgressStepsContainer() {
   return (
     <ProgressSteps current={current}>
       <Step title="Create Account">
-        <div className={css({...theme.typography.font300})}>
+        <div
+          className={css({
+            ...theme.typography.font200,
```

Comment:
We've been moving away from `fontx00` in examples and trying to include `paragraph`, `label`, etc typography styles shown on this page: https://baseweb.design/guides/theming/#typography. Can you please update this within the examples?

Reply:
Sure!